tests/metrics/test-irloc.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2317
+# TODO: REMOVE SHELLCHECK DISABLE RIGHT AFTER IMPLEMENTING irloc.sh
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2024 Yegor Bugayenko
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+set -e
+set -o pipefail
+
+# TODO: ENABLE THIS TESTS RIGHT AFTER IMPLEMENTING irloc.sh VIA REMOVING `exit 0`
+exit 0
+
+temp=$1
+stdout=$2
+
+{
+  tmp=$(mktemp -d /tmp/XXXX)
+  cd "${tmp}"
+  mkdir -p "${LOCAL}/${temp}"
+  touch "${LOCAL}/${temp}/file.java"
+  "${LOCAL}/metrics/irloc.sh" "${LOCAL}/${temp}/file.java" "${LOCAL}/${temp}/stdout"
+  grep "Provided non-git repo" "${LOCAL}/${temp}/stdout"
+} >"${stdout}" 2>&1
+echo "👍🏻 Didn't fail in non-git directory"
+
+{
+  tmp=$(mktemp -d /tmp/XXXX)
+  cd "${tmp}"
+  rm -rf ./*
+  rm -rf .git
+  git init --quiet .
+  git config user.email 'foo@example.com'
+  git config user.name 'Foo'
+  file1="one.java"
+  "${LOCAL}/metrics/irloc.sh" "./${file1}" "t0"
+  grep "File does not exist" "t0" # The given file does not exist
+} >"${stdout}" 2>&1
+echo "👍🏻 Didn't fail in repo without given file"
+
+{
+  tmp=$(mktemp -d /tmp/XXXX)
+  cd "${tmp}"
+  rm -rf ./*
+  rm -rf .git
+  git init --quiet .
+  git config user.email 'foo@example.com'
+  git config user.name 'Foo'
+  file1="one.java"
+  touch "${file1}"
+  "${LOCAL}/metrics/irloc.sh" "./${file1}" "t0"
+  grep "No commits yet in repo" "t0" # There are no commits in repo with given file
+} >"${stdout}" 2>&1
+echo "👍🏻 Didn't fail in repo without commits"
+
+{
+  tmp=$(mktemp -d /tmp/XXXX)
+  cd "${tmp}"
+  rm -rf ./*
+  rm -rf .git
+  git init --quiet .
+  git config user.email 'foo@example.com'
+  git config user.name 'Foo'
+  file1="one.java"
+  touch "${file1}"
+  git add "${file1}"
+  git commit --quiet -m "first file"
+  "${LOCAL}/metrics/irloc.sh" "./${file1}" "t0"
+  grep "Zero lines of code in repo" "t0" # There are only empty files committed in the repo
```

Comment:
@zaqbez39me The output should be `IRLoC 0` (see how other script print their results)